src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -17,49 +17,85 @@ def __init__(self):
     def set_screenshot_directory(self, path, persist=False):
         """Sets the root output directory for captured screenshots.
 
-        ``path`` argument specifies the absolute path where the screenshots should
-        be written to. If the specified ``path`` does not exist, it will be created.
-        Setting ``persist`` specifies that the given ``path`` should
-        be used for the rest of the test execution, otherwise the path will be restored
-        at the end of the currently executing scope.
+        ``path`` argument specifies the absolute path where the screenshots
+        should be written to. If the specified ``path`` does not exist,
+        it will be created. Setting ``persist`` specifies that the given
+        ``path`` should be used for the rest of the test execution, otherwise
+        the path will be restored at the end of the currently executing scope.
         """
         path = os.path.abspath(path)
         self._create_directory(path)
         if persist is False:
             self._screenshot_path_stack.append(self.screenshot_root_directory)
             # Restore after current scope ends
-            utils.events.on('scope_end', 'current', self._restore_screenshot_directory)
+            utils.events.on('scope_end', 'current',
+                            self._restore_screenshot_directory)
 
         self.screenshot_root_directory = path
 
     def capture_page_screenshot(self, filename=None):
         """Takes a screenshot of the current page and embeds it into the log.
 
-        `filename` argument specifies the name of the file to write the
-        screenshot into. If no `filename` is given, the screenshot is saved into file
-        `selenium-screenshot-<counter>.png` under the directory where
-        the Robot Framework log file is written into. The `filename` is
+        ``filename`` argument specifies the name of the file to write the
+        screenshot into. If no ``filename`` is given, the screenshot is saved
+        into file `selenium-screenshot-{index}.png` under the directory where
```

Comment:
It's good that single backticks have been replaced with double backticks in arguments because resulting formatting looks better and there's no risk for accidental linking. I think also `selenium-screenshot-{index}.png` should get the same treatment. Alternatively it could be formatted to be italics my surrounding it with `_`.